pkgs/os-specific/darwin/aldente/default.nix

```
@@ -0,0 +1,70 @@
+{ lib
+, stdenvNoCC
+, fetchurl
+, undmg
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "aldente";
+  version = "1.22";
+
+  src = fetchurl {
+    name = "AlDente.dmg";
+    url = "https://github.com/davidwernhart/aldente-charge-limiter/releases/download/${finalAttrs.version}/AlDente.dmg";
+    hash = "sha256-y8S9SCDbvoKGtNqGQvJDSpbLhh7GqXUo7pLcMzuCnjo=";
+  };
+
+  dontPatch = true;
+  dontConfigure = true;
+  dontBuild = true;
+  dontFixup = true;
+
```

```suggestion
  dontBuild = true;
  dontFixup = true;

```
patching and configure should be no-ops. Build maybe, too and is fixing up breaking things?